api-extractor/src/definitions/ApiDocumentation.ts

```diff
@@ -252,44 +253,27 @@ export default class ApiDocumentation {
     }
   }
 
-  constructor(apiItem: ApiItem,
+  constructor(docComment: string,
     docItemLoader: DocItemLoader,
     extractor: Extractor,
     errorLogger: (message: string) => void) {
-    this.apiItem = apiItem;
+    this.docComment = docComment;
     this.docItemLoader = docItemLoader;
     this.extractor = extractor;
     this.reportError = errorLogger;
-    this.docComment = this._getJsDocs(apiItem);
     this.parameters = new Map<string, IParam>();
     this._parseDocs();
   }
 
-  protected _getJsDocs(apiItem: ApiItem): string {
-    if (!apiItem.jsdocNode) {
-      return '';
-    }
-    const jsDoc: string = TypeScriptHelpers.getJsDocComments(apiItem.jsdocNode, this.reportError);
-
-    // Eliminate tags and then count the English letters.  Are there at least 10 letters of text?
-    // If not, we consider the definition to be "missingDocumentation".
-    const condensedDocs: string = jsDoc
```

Comment:
For isMissingDocs, we probably want something like this:
ApiItem.needsDocumentation   <--  we should warn the user "// (undocumented)"
ApiDocumentation.isDocumented  <-- the JSDoc doesn't have any comments, which may be the right thing or not